tests/helper/helper_generic.go

```diff
@@ -293,6 +295,9 @@ func CommonBeforeEach() CommonVar {
 	commonVar.Project = commonVar.CliRunner.CreateRandNamespaceProject()
 	commonVar.OriginalWorkingDirectory = Getwd()
 	os.Setenv("GLOBALODOCONFIG", filepath.Join(commonVar.Context, "preference.yaml"))
+	// Set ConsentTelemetry to false so that it does not prompt to set a preference value
+	cfg, _ := preference.New()
+	_ = cfg.SetConfiguration(preference.ConsentTelemetrySetting, "false")
```

Comment:
I think we should not ignore the error here and fail starting the CI job if there's failure doing this.